packages/api-reference/src/components/Layouts/Layouts.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+/**
+ * This component allows dynamic selection of various layout configurations
+ */
+import type { ReferenceLayoutProps, ReferenceLayoutSlots } from '../../types'
+import ClassicLayout from './ClassicLayout.vue'
+import ModernLayout from './ModernLayout.vue'
+
+// eslint-disable-next-line vue/no-unused-properties
```

Comment:
should we just pick the props we use from this?

Reply:
No we are passing props through.